reference/docs-conceptual/dsc/getting-started/lnxGettingStarted.md

```diff
@@ -13,7 +13,7 @@ The following Linux operating system versions are supported for DSC for Linux.
 
 - CentOS 5, 6, and 7 (x86/x64)
 - Debian GNU/Linux 6, 7 and 8 (x86/x64)
-- Oracle Linux 5, 6 and 7 (x86/x64)
+- Oracle Linux 6.8+ and 7.8+ (x86/x64)
```

Comment:
```suggestion
```
Oracle linux is not support

We should not duplicate the version list from the PowerShell LifeCycle page.

Reply:
In fact most of the list is wrong, or out of date.
https://docs.microsoft.com/en-us/powershell/scripting/powershell-support-lifecycle?view=powershell-7#supported-platforms